core/core-impl/src/main/java/ai/timefold/solver/core/impl/score/director/AbstractScoreDirectorFactory.java

```diff
@@ -68,6 +69,14 @@ public void setAssertClonedSolution(boolean assertClonedSolution) {
         this.assertClonedSolution = assertClonedSolution;
     }
 
+    public boolean isTrackingWorkingSolution() {
```

Comment:
This deserves a Javadoc, pointing to someplace that describes what it will do. Perhaps a reference to the relevant `EnvironmentMode`?